examples/crate_universe/has_aliased_deps/Cargo.toml

```diff
@@ -6,3 +6,4 @@ edition = "2018"
 
 [dependencies]
 reqwest = { version = "0.10.7", features = ["blocking", "json"] }
+indexmap = { version = "1.5.2", features = ["std"] }
```

Comment:
Why this change?

Reply:
I was running into the following error without it. 
```
error[E0107]: this struct takes 3 type arguments but only 2 type arguments were supplied
  --> external/has_aliased_deps_deps__h2__0_2_7/src/proto/streams/store.rs:15:10
   |
15 |     ids: IndexMap<StreamId, SlabIndex>,
   |          ^^^^^^^^ --------  --------- supplied 2 type arguments
   |          |
   |          expected 3 type arguments
   |
note: struct defined here, with 3 type parameters: `K`, `V`, `S`
help: add missing type argument
   |
15 |     ids: IndexMap<StreamId, SlabIndex, S>,
   |                                      ^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0107`.
```

I came across https://github.com/yewstack/yew/issues/1664 so decided to apply the same.

Reply:
Can you file an issue for this with repro steps? This shouldn't be necessary

Reply:
In attempting to do that, the issue seems to have vanished? Perhaps I had some bad combination of changes before I broke this out into it's own PR. I'll try to find what triggered it again but it now appears to genuinely be unnecessary.